tasks.py

```diff
@@ -143,6 +143,7 @@ def _detect_space(repo, branch=None, yes=False):
     ('prod', _detect_prod),
     ('stage', lambda _, branch: branch.startswith('release')),
     ('dev', lambda _, branch: branch == 'develop'),
+    ('dev', lambda _, branch: branch == 'feature/4511-add-TOS-and-AUP-to-api-key-signup'),
```

Comment:
We need to remove this line before the release

Reply:
@fec-jli @lbeaufort this was my mistake--the following PR resolves: https://github.com/fecgov/openFEC/pull/4536